src/Halogen/Portal.purs

```diff
@@ -50,6 +52,7 @@ type State query input output n
 -- ```
 --
 -- Another option is to use `H.hoist` to lift your component into `ReaderT`.
+newtype NT :: forall k. (k -> Type) -> (k -> Type) -> Type
```

Comment:
This is meant to be used with monads, so there's no need to be polymorphic in `k`.

```suggestion
newtype NT :: (Type -> Type) -> (Type -> Type) -> Type
```